executor/app.py

```
@@ -0,0 +1,504 @@
+            progress = 0
+
+        return TaskResponse(
+            task_id=task_id,
+            subtask_id=subtask_id,
+            status=status.value,
+            message=message,
+            progress=progress,
+        )
+
+    except Exception as e:
+        logger.exception(f"Error executing task {task_id}: {str(e)}")
```

Ruff (0.14.14)

Use explicit conversion flag

Replace with conversion flag

(RUF010)